### PR TITLE
Create community_repos.json

### PR DESCRIPTION
**Pull request by byrcolin**
This is to support the add gem repos in the project manger

### CODE REVIEW

File: src/community_repos.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"partner_repos": [
+		"https://overlo3de.com"
+	],
+	"community_repos": [
+
```
Comment (spham-amzn) [inline src/community_repos.json]: nit: empty space

**Review by spham-amzn — COMMENTED** [review]
The spacing/identing seems big, should keep it to 4-space indents for consistency

File: src/community_repos.json (hunk 1 above)
Comment (byrcolin) [reply src/community_repos.json]: done

**Comment by byrcolin** [comment]
fixed

**Review by spham-amzn — APPROVED** [review]

**Pull request merged**